packages/analytics/src/events.ts

```diff
@@ -11,11 +11,11 @@ interface HistoricalEvents {
   broadcast_ordinal_transaction: undefined;
   broadcast_ordinal_error: { error: any };
   broadcast_transaction: {
-    symbol: string;
-    amount: number;
-    fee: number;
-    inputs: number;
-    outputs: number;
+    symbol?: string;
```

Comment:
This looks a bit unusual. It seems like we do always need `symbol` for this? 

As it's for analytics maybe we can just have a more generic `Record` here?

We call this 3 times and only once does it have only `symbol` so maybe instead we should feed more parameters in that case? 

```
// use-submit-stx-transactions
 void analytics.track('broadcast_transaction', { symbol: 'stx' });
```

Reply:
Trying to just inline replace and not change much yet.